
syntax = "proto3";
package tag_recommend_system;

// 电影信息
message film_info {
  string film_id = 1;               // 电影名
  string web_id = 2;                // 网站名
  double web_score = 3;              // 网站的打分
  double web_full_score = 4;         // 网站的满分
  int32 date = 5;                  // 上映年份
  repeated string tags = 6;         // 类型
  string areas = 7;                 // 制作国家
  string directors_and_actors = 8;  // 导演、演员
  string quote = 9;                // 短评
}

message tag_films {
  // 一个tag对应的所有电影信息
  repeated film_info film_infos = 1;
}
